src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "AppInstallerStrings.h"
+#include "winget/settings/Source.h"
+#include "winget/settings/Visual.h"
+#include <json.h>
```

Comment:
A consumer of this header should not need to include JSON.  That should all be implementation in the cpp file.